.github/workflows/python-build-rest.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python Build

```

Comment:
Could you change the name of either this workflow or the other? Maybe name this Python Build Rest and leave the other one as Python Build or leave this as Python Build and name the other workflow lichess-bot Build?